packages/alea-frontend/pages/api/get-section-info/[courseId].ts

```diff
@@ -98,7 +98,7 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     notesFilepath
   );
   const allSections = getAllSections(docSections) as SectionInfo[];
-  const coverageData = getCoverageData()[courseId];
+  const coverageData = getCoverageData()[courseId].filter(snap=>snap.sectionName);
```

Comment:
Looks like you forget to format your code or prettier is not on be default. Please connect with Rakesh to know more.